docs/api.md

```diff
@@ -325,6 +326,11 @@ Request a verification code to be issued. Accepts [optional] symptom date and te
     the caller should apply a cryptographic hash before sending that data. **The
     system does not sanitize or encrypt these external IDs, it is the caller's
     responsibility to do so.**
+* `onlyGenerateSMS` is an optional field. If supplied, the system will **not**
```

Comment:
also note that the phone number still needs to be passed in for the authenticated SMS piece.